std/manual.md

```diff
@@ -951,6 +951,51 @@ for await (const req of serve(":8000")) {
 $ deno run --importmap=import_map.json hello_server.ts
 ```
 
+## WASM support
+
+Deno can import a [wasm](https://webassembly.org/) binary.
```

Comment:
I think we should mention that Deno supports the normal way of loading WASM too 
https://github.com/denoland/deno/blob/master/cli/tests/wasm.ts

Reply:
Thanks for fixing.